msg/airspeed_validated.msg

```diff
@@ -1,12 +1,12 @@
-uint64 timestamp			# time since system start (microseconds)
+uint64 timestamp				# time since system start (microseconds)
 
-float32 indicated_airspeed_m_s		# indicated airspeed in m/s (IAS), set to NAN if invalid
-float32 equivalent_airspeed_m_s     # equivalent airspeed in m/s (accounts for instrumentation errors) (EAS), set to NAN if invalid
+float32 indicated_airspeed_m_s			# indicated airspeed in m/s (IAS), set to NAN if invalid
+float32 calibrated_airspeed_m_s     		# calibrated airspeed in m/s (CAS, accounts for instrumentation errors), set to NAN if invalid
 float32 true_airspeed_m_s			# true filtered airspeed in m/s (TAS), set to NAN if invalid
 
-float32 equivalent_ground_minus_wind_m_s # EAS calculated from groundspeed - windspeed, where windspeed is estimated based on a zero-sideslip assumption, set to NAN if invalid
+float32 calibrated_ground_minus_wind_m_s 	# CAS calculated from groundspeed - windspeed, where windspeed is estimated based on a zero-sideslip assumption, set to NAN if invalid
 float32 true_ground_minus_wind_m_s 		# TAS calculated from groundspeed - windspeed, where windspeed is estimated based on a zero-sideslip assumption, set to NAN if invalid
 
-bool airspeed_sensor_measurement_valid # True if data from at least one airspeed sensor is declared valid.
+bool airspeed_sensor_measurement_valid 		# True if data from at least one airspeed sensor is declared valid.
 
-int8 selected_airspeed_index # 1-3: airspeed sensor index, 0: groundspeed-windspeed, -1: airspeed invalid
+int8 selected_airspeed_index 			# 1-3: airspeed sensor index, 0: groundspeed-windspeed, -1: airspeed invalid
```

Comment:
This indexing is kind of confusing, something we should revisit later.

Reply:
Agree, but let's first align our next steps concerning multi airspeed/mulit EKF.